test/powershell/Modules/Microsoft.PowerShell.Management/Split-Path.Tests.ps1

```diff
@@ -29,6 +29,10 @@ Describe "Split-Path" -Tags "CI" {
 	{ Split-Path -Qualifier -ErrorAction Stop abcdef } | Should -Throw
     }
 
+    It "Should error given positional parameter #2" {
+	{ Split-Path env: $NULL } | Should -Throw
+    }
```

Comment:
Question for reviewers: do we really need the test?

Reply:
Normally, I'd say no... but this would be a breaking change if we take it. So in that sense I think it's safer to have the test, so that if this changes again/regresses at some point it's more visible.

Reply:
In the case a comment would be good why we added the test.